libraries/lib-sentry-reporting/SentryReport.cpp

```diff
@@ -38,6 +39,19 @@ namespace sentry
 {
 namespace
 {
+
+//! Remove paths from the log    
+std::string AnonymizePaths(const std::string& input)
+{
+   // Finding the path boundary in the arbitrary text is a hard task.
+   // We assume that spaces cannot be a part of the path. 
+   // In the worst case - we will get <path> <path> 
+   // Also, we assume, that
```

Comment:
Did you omit part this comment?

Reply:
And I no longer remember, what I've planned to write anyway. I will clean it up